python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -11,39 +11,46 @@
 from DIRAC.Interfaces.API.DiracAdmin import DiracAdmin
 dirac = Dirac()
 
-# Write to output pipe
-#/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
-#reserved_stdout = sys.stdout
-#sys.stdout = sys.stderr
-# def output(object):
-#    reserved_stdout.write("CHECK ME " + str(object))
-#    reserved_stdout.write(pickle.dumps(object))
-
 # Dirac commands
 #/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
 
-def getJobGroupJobs(jg): output(dirac.selectJobs(jobGroup=jg))
+def getJobGroupJobs(jg):
+    ''' Return jobs in a group'''
```

Comment:
[PEP 257](https://www.python.org/dev/peps/pep-0257/#id15) suggests triple double quotes for docstrings.